stats/stats.go

```diff
@@ -0,0 +1,31 @@
+package services
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-cli-artifactory/stats"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-cli/utils/cliutils"
+	"github.com/urfave/cli"
+)
+
+func GetStats(c *cli.Context) error {
+	format := c.String("format")
+	accessToken := c.String("access-token")
+	serverId := c.String("server-id")
+	if c.NArg() != 1 {
+		_ = cli.ShowSubcommandHelp(c)
+		return cliutils.WrongNumberOfArgumentsHandler(c)
+	}
+	productName := c.Args().First()
+	if productName == "rt" || productName == "artifactory" {
+		newStatsCommand := stats.NewStatsCommand().
+			SetAccessToken(accessToken).
+			SetServerId(serverId).
+			SetFormat(format)
+		return newStatsCommand.Run()
+	} else {
+		_ = cli.ShowSubcommandHelp(c)
+		coreutils.ExitOnErr(errors.New("Wrong product " + productName))
```

Comment:
you dont have to exit here return the error
```suggestion
		return fmt.Errorf("wrong product %s only artifactory or rt is supported", productName))
```